testing/e2e/workload_test.go

```diff
@@ -392,23 +392,6 @@ func TestCreateFromGitWithAnnotations(t *testing.T) {
 				},
 			},
 		},
-		{
-			Name:         "Get the updated workload",
-			WorkloadName: "test-create-git-annotations-workload",
-			Command: *it.NewTanzuAppsCommandLine(
-				"workload", "get", "test-create-git-annotations-workload", namespaceFlag),
-			Verify: func(t *testing.T, output string, err error) {
-				if regexPod.FindString(output) == "" {
-					t.Errorf("expected Pod results in output %v", output)
-					t.FailNow()
-				}
-				decodedString := strconv.QuoteToASCII(output)
```

Comment:
when you removed this test, remove `strconv` as dep as well. tests are failing because of that